Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -540,41 +540,54 @@ define([
      */
     GlobeSurfaceTileProvider.prototype.computeDistanceToTile = function(tile, frameState) {
```

Comment:
@lilleyse we want this fix in the 3d-tiles branch `TileBoundingBox` class, otherwise we will reintroduce this bug when merging `TileBoundingBox` to master.  You might want to cherry pick these commits and make the change right away.